go-controller/pkg/cluster/node.go

```diff
@@ -81,6 +81,9 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	var cidr string
 	var wg sync.WaitGroup
 
+	var LogLevel = logrus.GetLevel()
```

Comment:
@pecameron could you add a code comment just quickly saying that why we're doing this? Otherwise lgtm.

Reply:
done